### PR TITLE
added hw-32-1

### PR DESCRIPTION
**Pull request by kenu21**

### CODE REVIEW

File: src/main/java/mate/academy/spring/controller/BookController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    @GetMapping
+    public String getAllBooks(ModelMap model) {
+        model.put("allBooks", bookService.listBooks());
+        return "allBooks";
+    }
+
+    @GetMapping("/info")
+    public String bookInfo(ModelMap model, @RequestParam Long id) {
+        model.put("infoBook", bookService.get(id).orElseThrow(NoSuchElementException::new));
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/controller/BookController.java]: better change orElseThrow(NoSuchElementException::new) to rediection on exception page

File: src/main/java/mate/academy/spring/controller/InjectController.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/inject")
+public class InjectController {
+
+    @GetMapping
+    public String injectData() {
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/controller/InjectController.java]: optional: we can use annotation PostConstruct for injecting data when we start the application

Comment (kenu21) [reply src/main/java/mate/academy/spring/controller/InjectController.java]: I'd like to have choice: inject data or no.

File: src/main/java/mate/academy/spring/controller/BookController.java (hunk 1 above)
Comment (kenu21) [reply src/main/java/mate/academy/spring/controller/BookController.java]: OK.

**Pull request merged**